src/backend/ze/pup/yaksuri_zei_get_ptr_attr.c

```diff
@@ -9,6 +9,29 @@
 #include "yaksuri_zei.h"
 #include <level_zero/ze_api.h>
 
+static inline int find_device_num(ze_device_handle_t device)
+{
+    int device_num = -1;
+    for (int i = 0; i < yaksuri_zei_global.ndevices; i++)
+        if (yaksuri_zei_global.device[i] == device) {
+            device_num = i;
+            break;
+        }
+    if (device_num == -1) {
+        /* subdevice */
+        ze_device_properties_t subdeviceProps;
+        zeDeviceGetProperties(device, &subdeviceProps);
+        for (int i = 0; i < yaksuri_zei_global.ndevices; i++) {
+            yaksuri_zei_device_state_s *device_state = yaksuri_zei_global.device_states + i;
+            if (subdeviceProps.deviceId == device_state->deviceId) {
+                device_num = i;
+                break;
+            }
+        }
+    }
+    return device_num;
+}
+
```

Comment:
For commit "backend/ze: support for tiles in an Intel GPU device. ", Please add commit message explain what are tiles and what does this commit adds.

Reply:
Tile is a subdevice.